rosidl_generator_cpp/rosidl_generator_cpp/__init__.py

```diff
@@ -176,7 +176,106 @@ def primitive_value_to_cpp(type_, value):
     assert False, "unknown primitive type '%s'" % type_.type
 
 
+def default_cpp_value_from_type(type_):
+    if type_ == 'string':
+        return ''
+    elif type_ in ['float32', 'float64']:
+        return '0.0'
+    elif type_ == 'bool':
+        return 'false'
+    return '0'
```

Comment:
Do unsigned types need a trailing `u`?

Reply:
Same as above, `primitive_value_to_cpp` will handle this.